### DISCUSSION TITLE
create vars globals

### DISCUSSION THREAD

**Discussion by hegog25** [jsightapi/online-editor-frontend#220] [Ideas]
Is posible for new features create vars globals that change of state and use in any part of comments?

**Comment by andrew-platonov** [comment]
Hello! I do not quite understand, what do you mean? Can you explain by a short example?

**Reply by hegog25** [reply]
ok, example: 
```
TYPE @KO_404 // A common error response with code 404. Resource  ${resource}
{
  "success": false,
  "content": {
    "code": 404,
    "name": "ResourceNotFound",
    "message": "${resource}, not found"
  }
}
```

This type in the message key and in comment use ${resource}, is posible set argument for replace this value?
